direction/index.md

```diff
@@ -1,42 +1,60 @@
 
-Improving upon on our [basic code intelligence](https://docs.sourcegraph.com/user/code_intelligence) that works for every language, Sourcegraph provides precise code intelligence for a subset of common languages including Go, TypeScript, C/C++, Java, and C#.
+### Support a broader set of configurations and workflows
 
-### Enhanced scalablity, reliability, and security
+#### Challenge to overcome
 
-Sourcegraph performs at the scale of our largest customers under a wide variety of configurations and deployment infrastructure. [Code search is fast at large scale](https://docs.google.com/document/d/18w8T_KzYxQye8wg1g01QpMOX4_ERTtbOxMBRYaOEkmk/edit?usp=sharing) (~80k repositories), and the API can support enterprise level usage. Sourcegraph enforces repository permissions using ACLs from Bitbucket Server, GitHub, and GitLab. Admins understand the health of their instances and are alerted proactively if things fail.
+Many organizations have unique code setups and workflows that require workarounds to support, or prevent them from using Sourcegraph entirely.
 
-### Code change management
+#### What we want to achieve
 
-Use Sourcegraph to [run large-scale code change campaigns](https://about.sourcegraph.com/product/code-change-management) to remove legacy code, fix critical security issues, and pay down tech debt. You can create campaigns across thousands of repositories and code owners. Sourcegraph automatically creates and updates all of the branches and pull requests, and you can track progress and activity in one place.
+Sourcegraph is the code search solution for every developer and every organization, regardless of their setup. At any organization, Sourcegraph is how developers are able to work with large and complex development workflows. Sourcegraph sustainably supports configuration of unique infrastructure and code setups.
+
+#### How we're going to do this
+
+- Solve use cases that are present at many large enterprises.
+- Reduce friction in getting Sourcegraph deployed at scale.
+- More integrated support for non-Git version control systems.
+
+### Surface insights into code and development
+
+#### Challenge to overcome
+
+Software development executives have a hard time understanding what is going on with their large teams, the projects they are investing in, and the impact developers and teams have inside of their organization. The insights and information they need is hard to come by and is not solved by other tools or services because it relies on data the development teams produce to be tied to higher order concerns.
+
+#### What we want to achieve
+
+Sourcegraph has access to the low level data needed to build insights that are high quality and meaningful. Sourcegraph feeds the symbiotic relationship between developer produced data and executive insights, creating a cyclical feedback loop of immense value. Sourcegraph is the place engineering leaders turn to understand what is going on in their organization.
```

Comment:
```suggestion
Sourcegraph has access to the low level data needed to build insights that are high quality and meaningful. Sourcegraph feeds the symbiotic relationship between developer produced data and executive insights, creating a cyclical feedback loop of immense value. Engineering leaders turn to Sourcegraph to understand what is going on in their organization.
```

I think the original is missing something here (phrasal verb "turn to" + non-finite "to understand")?